src/Bundle/Resources/config/services/controller.xml

```diff
@@ -15,6 +15,14 @@
     <services>
         <defaults public="true" />
 
+        <service id="sylius.symfony.main_controller" class="Sylius\Resource\Symfony\Controller\MainController" public="false">
```

Comment:
I would go with:
```suggestion
        <service id="sylius.main_controller" class="Sylius\Resource\Symfony\Controller\MainController" public="false">
```
or
```suggestion
        <service id="sylius.controller" class="Sylius\Resource\Symfony\Controller\MainController" public="false">
```

that should be enough

Reply:
:+1:  for `sylius.main_controller`. Service definition already in SF context, so no need to specify it explicitly.